packages/zambdas/src/ehr/change-in-person-visit-status/helpers/helpers.ts

```diff
@@ -115,6 +115,21 @@ const getUpdateInPersonEncounterStatusOperation = async (
 
   const encounterPatchOps: Operation[] = [{ op: 'replace', path: '/status', value: encounterStatus }];
 
+  if (['discharged', 'ready for discharge'].includes(updatedStatus)) {
+    const atndIndex = encounter.participant?.findIndex(
```

Comment:
while this will not come up as a typo using the spell checker because i added `atnd` as an exception, it's still a >3 character word that isn't in the dictionary. i'm guessing it's a short form of "attendant"? if so please use the full word